docs/developer-guide/get-started/emt-build-and-deploy.md

```diff
@@ -0,0 +1,373 @@
+# Build Edge Microvisor Toolkit and Deploy on Edge Nodes
+
+This guide will walk you through the entire process of deploying Edge Microvisor Toolkit.
+You will learn how to build the toolkit, install it on single or multiple edge node
+clusters, and deploy AI solutions.
+
+The Edge Microvisor Toolkit Standalone Node supports different EMT images to
+meet specific edge deployment needs. You can choose between:
+
+- **Edge Microvisor Toolkit Non-Real-Time image** (default)
+- **Edge Microvisor Toolkit Desktop Virtualization image**
+- **Custom immutable Edge Microvisor Toolkit created using** [Build Instructions](#1-build-the-microvisor)
+
+**Table of contents:**
+
+1. [Build a custom image of Edge Microvisor Toolkit.](#1-build-the-microvisor)
+2. [Create a bootable USB installer.](#2-create-a-bootable-usb-installer)
+3. [Install the toolkit on an edge device.](#3-install-the-toolkit-on-an-edge-device)
+4. [Deploy AI solution.](#4-deploy-ai-solution)
+
+## 1. Build the microvisor
+
+If you want to create your own custom standalone node, follow the instructions below to build
+the microvisor from scratch. If you wish to use the default, non-real-time image of Edge Microvisor
+skip to [Create a bootable USB installer](#2-create-a-bootable-usb-installer).
+
+1. Clone the stable branch of edge-microvisor-toolkit repository:
```

Comment:
Perhaps, state as an example using a tagged version such as, as this will change on a week by week basis. Perhaps tell the user how to list all the tagged releases and pick the latest one?